src/styles/utils/_variables.scss

```
@@ -0,0 +1,6 @@
+$color-blue--light: #34568b;
+$color-blue--dark: #00acdc;
+$color-gray: #ccc;
+$margin-big: 48px;
+$padding-big: 50px;
+$transition: 300ms;
```

```suggestion
$transition-time: 300ms;
```
It's better to specify